PACKAGE_NAME not declared error building on netbeans

**Synfig version & platform**:
master branch
commit e33a998

Debian unstable

**Issue description**:
I followed setting [NetBeans environment for synfig](https://synfig-docs-dev.readthedocs.io/en/latest/common/Setting%20up%20netbeans%20IDE.html) and did my first build.
Below was the error.

> /home/keyikedalube/Source/synfig/synfig-studio/src/gui/dialogs/about.cpp:83:19: error: ‘PACKAGE_NAME’ was not declared in this scope
>    83 |  set_program_name(PACKAGE_NAME);
>       |                   ^~~~~~~~~~~~
> make[2]: *** [synfig-studio/src/gui/CMakeFiles/synfigstudio.dir/build.make:557: synfig-studio/src/gui/CMakeFiles/synfigstudio.dir/dialogs/about.cpp.o] Error 1
> make[1]: *** [CMakeFiles/Makefile2:1501: synfig-studio/src/gui/CMakeFiles/synfigstudio.dir/all] Error 2
> make: *** [Makefile:130: all] Error 2

I tried building from a clean folder on second try. Got the same error.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/87532478-package_name-not-declared-error-building-on-netbeans?utm_campaign=plugin&utm_content=tracker%2F321991&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F321991&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/synfig-studio/src/gui/dialogs/about.cpp
+++ b/synfig-studio/src/gui/dialogs/about.cpp
@@ -59,6 +59,10 @@ using namespace studio;
 
 /* === M A C R O S ========================================================= */
 
+#ifndef PACKAGE_NAME
+#define  PACKAGE_NAME "Synfig Studio"
+#endif
+
 #ifndef VERSION
 #define VERSION	"unknown"
 #endif

